docs/installation_guide.md

```diff
@@ -166,84 +241,145 @@ uninstall_all(){
+kubectl create secret generic rabbit --from-literal=password=${pw} -n api_gateway
+kubectl create secret generic rabbit --from-literal=password=${pw} -n uva
+kubectl create secret generic rabbit --from-literal=password=${pw} -n vu
+kubectl create secret generic rabbit --from-literal=password=${pw} -n surf
+#  If there are any new namespaces, add them here
+
+# Hash password 
+docker run --rm  rabbitmq:3-management rabbitmqctl hash_password $pw
 ```
+**Important**: Save the password we are going to use it in the next step!
 
 ## Create definitions file
+From the root directory of DYNAMOS, create a copy of the definitions file like so:
 ```bash
 cp configuration/k8s_service_files/definitions_example.json configuration/k8s_service_files/definitions.json
 ```
-Update the docker password in the definitions file
+Update the docker password in the definitions file with the $pw we created in the previous step
+
+## Configure Rabbit PVC
 ```bash
-Send an email to create Jorrit to get the password
+cd configuration
+./fill-rabbit-pvc.sh
 ```
 
 ## Update hostfile
-```
+To be able to access DYNAMOS from your local machine, you'll need to add the `api-gateway` service to your hosts file.
+To do this on Linux, use your favourite text editor with root access on the file `/etc/hosts`, like so:
+```bash
 sudo vim /etc/hosts
+```
+Now add the following to hosts file:
+```bash
 127.0.0.1 api-gateway.api-gateway.svc.cluster.local
 ```
+Since the API gateway is the only public facing service, it is the only entry required in the hosts file. If any additional services are added, they should also be added here with a similiar pattern.
 
-## Add ingress
-```bash
-helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+Note that this is super useful when trying to test DYNAMOS locally using tools such as `curl` or `postman`.
+
+## Example Request
+To make sure we installed everything properly, let's use the AMDeX use case as an example.
+Firstly, make sure you've deployed everything, you can do that using `deploy_all()` in your CLI, or individually deploying each service and checking the status on k9s or using the `watch_pods()` method.
+
+Let's setup the request.
+
+The URL should be:
+```
+http://api-gateway.api-gateway.svc.cluster.local:32093/api/v1/requestApproval
+```
+as a **POST** request, with the following body with **JSON** encoding:
+```json
+{
+    "type": "sqlDataRequest",
+    "user": {
+        "id": "12324",
+        "userName": "jorrit.stutterheim@cloudnation.nl"
+    },
+    "dataProviders": ["VU","UVA","RUG"],
+    "data_request": {
+        "type": "sqlDataRequest",
+        "query" : "SELECT * FROM Personen p JOIN Aanstellingen s LIMIT 1000",
+        "algorithm" : "average",
+        "options" : {
+            "graph" : false,
+            "aggregate": false 
+        },
+        "requestMetadata": {}   
+    }
+}
 ```
 
-## Deploy Ingress
+## Delete the rabbit secrets
```

Comment:
This is strange at the end, since this should be done after deploying the namespaces if the previously mentioned script is not used. 

So I think this should be removed, and unified with the process to the proper deploy namespaces command, with generated password be